.eslintrc.js

```diff
@@ -111,8 +117,6 @@ module.exports = {
         ],
       },
     ],
-    'import/extensions': ['error', 'ignorePackages'],
```

Comment:
import rules were not actually working without a resolver. This complains about the missing ts extension everywhere (we can re-add it later)